ultraplot/axes/geo.py

```diff
@@ -480,6 +501,255 @@ def __init__(self, *args, **kwargs):
         """
         super().__init__(*args, **kwargs)
 
+    @override
+    def _sharey_limits(self, sharey: "GeoAxes"):
+        return self._share_limits(sharey, which="y")
+
+    @override
+    def _sharex_limits(self, sharex: "GeoAxes"):
+        return self._share_limits(sharex, which="x")
```

Comment:
The lack of code coverage here means we need an additional test or there is a bug.

Reply:
Fixed!